datasetinsights/io/downloader/unity_simulation.py

```diff
@@ -0,0 +1,120 @@
+import logging
+import os
+import re
+
+from datasetinsights.io.downloader.base import DatasetDownloader
+from datasetinsights.io.usim import Downloader, download_manifest
```

Comment:
How is `io.usim.Downloader` different from `UnitySimulationDownloader`? Should we rename module or move file around to make it easier to understand? Thoughts?

Reply:
I would move the code into the UnitySimulationDownloader. As its not different in the usecase

Reply:
This will require even more refactoring as the Downloader class is used in synthtic.py and statistics.py

Reply:
Both ` synthtic.py` and `statistics.py` should not have logic to download dataset. They should be simply loading data (assuming dataset exists in local folder)

Reply:
I know they should not have the logic but they do currently. I'm just trying to point out that this increases the scope of the PR.